common/http/headers_test.go

```diff
@@ -11,9 +11,6 @@ import (
 
 const uuid_regex = `^[[:xdigit:]]{8}(-[[:xdigit:]]{4}){3}-[[:xdigit:]]{12}$`
 
-// 64-bit random hexadecimal string
-const b3_id_regex = `^[[:xdigit:]]{16}$`
```

Comment:
not used anywhere (and it is private)

Reply:
Great. Please try to do that in a separate commit next time.